test/cpos/test_cpo_schedule.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Lucian Radu Teodorescu
+ *
+ * Licensed under the Apache License Version 2.0 with LLVM Exceptions
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *   https://llvm.org/LICENSE.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <catch2/catch.hpp>
+#include <execution.hpp>
+
+namespace ex = std::execution;
+
+struct my_sender {
+  template <template <class...> class Tuple, template <class...> class Variant>
+  using value_types = Variant<Tuple<>>;
+  template <template <class...> class Variant>
+  using error_types = Variant<std::exception_ptr>;
+  static constexpr bool sends_done = true;
```

Comment:
These days I would use `completion_signatures` for this.
```suggestion
struct my_sender
  : std::execution::completion_signatures<
      std::execution::set_value_t(),
      std::execution::set_error_t(std::exception_ptr),
      std::execution::set_done_t()> {
```

Reply:
Didn't know about this. Thank you